include/userobjects/MyTRIMRasterizer.h

```diff
@@ -85,9 +85,15 @@ class MyTRIMRasterizer : public ElementUserObject
     /// the TRIM class to instantiate in the recoil loops
     TRIMModuleEnum trim_module;
 
-    ///energy cutoff below which recoils are not followed explicitly but effects are calculated analytically
+    /// energy cutoff below which recoils are not followed explicitly but effects are calculated analytically
     Real analytical_cutoff;
 
+    /// enable instantaneous recombination
+    bool recombination;
```

Comment:
where is the leading `_` => `_recombination`

Reply:
I'm moving away form leading underscores in simple struct members. The benefit you have in classes with member functions is not there. I'll also go over MyTRIM and strip them off again. It turns out symbols that start with an underscore and are followed by a capital letter are reserved for internal implementation details.